### ISSUE TITLE
Unable to find file /etc/nv_tegra_release

### ISSUE THREAD

**Issue by DenisVoloshin** [jetsonhacks/jetsonUtilities#4]
When run `py jetsonInfo.py` on TX2, getting
Unable to find file /etc/nv_tegra_release

What I'm missing 

Thanks

**Comment by jetsonhacks** [comment]
I don't know what the command 'py' does. 
Does the file /etc/nv_tegra_release exist on your system?

**Comment by DenisVoloshin** [comment]
sorry, my typo, I meant` python jetsonInfo.py`, tried `./jetsonInfo.py`
Seems like /etc/nv_tegra_release doesn't exit, I check it manually by looking in the etc folder and by
cat: /etc/nv_tegra_release: No such file or directory

**Comment by erbornemeier** [comment]
```
 NVIDIA Jetson TX1
Error: Unable to find L4T Version
Reason: Unable to find file /etc/nv_tegra_release
 Ubuntu 18.04.3 LTS
 Kernel Version: 4.9.140-tegra
 CUDA 10.0.326
```

I get the above output when running the jetsonInfo.py file, the file does not exist

**Comment by jetsonhacks** [comment]
This should be fixed in release v2.0
